src/client/components/MealsWithId.js

```diff
@@ -0,0 +1,152 @@
+
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import "./home.css";
+
+export const MealsWithId = () => {
+    const { id } = useParams();
+    const [availableReservation, setAvailableReservation] = useState([]);
+
+    const [title, setNewTitle] = useState([]);
+    const [success, setSuccess] = useState("");
+    const [error, setError] = useState("");
+
+    // controlled inputs
+    const [phone, setPhone] = useState('');
+    const [name, setName] = useState('');
+    const [email, setEmail] = useState('');
+    const [seats, setSeats] = useState('');
+
+    useEffect(() => {
+        fetchData();
+    }, []);
+
+    const fetchData = async () => {
+        const result = await fetch(`http://localhost:3000/api/meals/${id}`);
+        const fetchResult = await result.json();
+        setNewTitle(fetchResult);
+    };
+
+    useEffect(()=>{
+    const result = fetch(`http://localhost:3000/api/meals?availableReservations`)
+    .then(res=>res.json())
+    .then(data=>setAvailableReservation(data));
+  },[]);
```

Comment:
Why do you use first `async/await` and after use `then()` ? it is better to stick with one and be consistent